dev.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
```

Comment:
I just realized we might not need this file since we can run `npm run serve` to view and modify the components. Is my understanding correct?

Reply:
That is correct. The purpose of the file was to emulate a kind of CDN experience and how to include each component individually when the build command is run.